test/unit/lodash.spec.js

```diff
@@ -0,0 +1,158 @@
+'use strict';
+
+var chai = require('chai');
+var expect = chai.expect;
+var _ = require('lodash');
+var datasets = require('../../app/datasets');
+
+describe('Lodash training', function()
+{
+
+    describe('constant', function()
+    {
+        var params = datasets.constant();
+        it('should check if first parameter is an object',function()
+        {
+            expect(params[0]).to.be.an('object');
+        });
+        it('should check objects values',function()
+        {
+            expect(params[0]).to.have.property('fred');
+            expect(params[0]).to.have.property('pebbles');
+            expect(params[0]).to.have.deep.property('john.age', 36);
+        });
+        it('should check if function returns an object', function(){
+             var getter = _.constant(params[0]);
+             expect(getter()).to.be.an('object');
+        });
+        it('should create a function that returns value', function(){
+                expect(_.constant.apply(_, params)).to.be.a('function');
+        });
+    });
+
+    describe('conforms', function()
+    {
+        var params = datasets.conforms();
+        var fun = _.conforms({ 'number': _.partial(_.gt, _, 38) });
+
+        it('should check if first parameter is an object',function()
+        {
+           expect(params[0]).to.be.an('object');
+        });
+        it('should check objects values',function()
+        {
+           expect(params[0]).to.have.property('day');
+           expect(params[0]).to.have.property('number',1);
+        });
+        it('should check if it returns a function', function(){
+            expect(fun).to.be.a('function');
+        });
+        it('Creates a function that invokes the predicate properties of source with the corresponding property values of a given object, returning true if all predicates return truthy, else false.', function()
+        {
+            expect(fun()).to.be.false;
+        });
+    });
+
+    describe('flow', function ()
+    {
+        var params = datasets.flow();
+        it('should check if first parameter is a function',function()
+        {
+           expect(params[0]).to.be.a('function');
+        });
+        it('should create a function that returns value.', function()
+        {
+            expect(_.flow.apply(_, params)).to.be.a('function');
+        });
+        it('should create a function that returns the result of invoking the given functions with the this binding of the created function, where each successive invocation is supplied the return value of the previous.', function (){
```

Comment:
This text in `it` doesn't say anything about your expections in test. It is just copied text from lodash documentation witch user will read there, not in test.